.gitignore

```diff
@@ -1,5 +1,6 @@
 # Environment variable files:
 .env
+**/.DS_Store
```

Comment:
This should go in your global ignore file because it isn't specific to this project, it's specific to your machine.

You can do this by creating `~/.config/git/ignore` with the files that are specific to your setup. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer.

Reply:
Thanks for the documentation! I have made this change.